coding-concepts.js

```diff
@@ -37,23 +38,23 @@ const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
 // console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: The expected output should be 'Javascript'
+// b) Verify and explain: I made the same mistake as in the case above. I need to make a point to remember the values begin at 0, not 1.
 
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
 // console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: The output will be 'Saturday Sunday'. .toUppercase changes the strings within the called on variable to change the beginning letter to upper case.
+// b) Verify and explain: I received an error message saying that .toUpperCase is not a function. The reason that it did not run is because .toUpperCase is specifically a built-in String Method within Javascript
 
 
 // --------------------5) What will this log?
 
 const dataTypes = ["number", "string", "Boolean"]
 // console.log(typeof dataTypes.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:I think the output is going to be 'number' but I'm not entirely certain why. .length would normally give the number of indexes within the given array or string. I'm not sure what typeof does or will do.
+// b) Verify and explain:I was correct, the given output was 'number', but I still did not understand why. The reason is that typeof is asking for what kind of data type is stored within the given variable, in this instance dataTypes. .length is getting the number of indexes (3) which is then being identified as a number by the typeof operator.
```

Comment:
Excellent description on what is happening.  Since you are checking the length of the array dataTypes, you will receive the number 3, but the typeof operator will check the data type of whatever follows it.